spotipy/util.py

```diff
@@ -118,15 +137,34 @@ def get_host_port(netloc):
 
 
 def normalize_scope(scope):
+    """Normalize the scope to verify that it is a list or tuple.
+
+    If the input is a string, the function splits the string by commas to
+    create a list of scopes.
+    If the input is already a list or tuple. it is used directly.
+
+    Args:
+        scope: A string representing scopes separated by commas,
+        or a list/tuple of scopes.
+
+    Returns:
+        A string of sorted scopes separated by spaces if
+        input is valid, otherwise None.
+
+    Raises:
+        Exception: If the scope is not a string, list, or tuple.
+        The scope value must be a list of scopes or a string of scopes
+        separated by commas.
+    """
     if scope:
         if isinstance(scope, str):
             scopes = scope.split(',')
         elif isinstance(scope, list) or isinstance(scope, tuple):
             scopes = scope
         else:
             raise Exception(
-                "Unsupported scope value, please either provide a list of scopes, "
-                "or a string of scopes separated by commas"
+                "Unsupported scope value, please either provide"
+                "a list of scopes, or a string of scopes separated by commas"
```

Comment:
Can you explain why you've changed this part? I don't really see any improvements and a space char is also missing now.

Reply:
This was due to a linting error that was flagged on my end and after reverting it, it does not seem like the linting error was raised. It is back to what it was before.